Wikipedia/UI-V5/WMFContinueReadingCell.m

```diff
@@ -0,0 +1,25 @@
+
+#import "WMFContinueReadingCell.h"
+
+@implementation WMFContinueReadingCell
+
+- (UICollectionViewLayoutAttributes*)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes*)layoutAttributes {
+    
+    CGFloat const preferredMaxLayoutWidth = layoutAttributes.size.width - self.leadingConstraint.constant - self.trailingConstraint.constant;
+
+    self.title.preferredMaxLayoutWidth = preferredMaxLayoutWidth;
+    self.summary.preferredMaxLayoutWidth = preferredMaxLayoutWidth;
+
+    //HACK: can't call [self.title systemLayoutSizeFittingSize:CGSizeMake(preferredMaxLayoutWidth, CGFLOAT_MAX)];
+    // This causes an arithmetic crash from some sort of semi-infinite loop through this method
```

Comment:
if the above works, we could refactor the "expand height to fit `width`" logic into a `UIView` category in the app, which the `VisualTestSizingUtils` class reuses